src/org/ods/services/OpenShiftService.groovy

```diff
@@ -804,7 +804,7 @@ class OpenShiftService {
      */
     String patch(String resource, Map<String, ?> patch, String path = null, String project = null) {
         if (!resource) {
-            throw new IllegalArgumentException('You must specify the resource to path')
+            throw new IllegalArgumentException('You must specify the resource to patch') //path or patch??
```

Comment:
@jchaves patch seems to be correct

Reply:
I thought so. New commit clearing up.

cheers!